bitcoin-extensions/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+	"manifest_version": 2,
+    "name": "Bitcoin Chrome Extension",
+    "version": "1.0",
+	"description": "This is the description for your first chrome extension",
+	"content_scripts": [
+		{
+			"matches": ["https://www.google.com/*"],
+			"css": ["styles.css"],
+			"js": ["scripts.js"]
+		}
+	],
+	"browser_action": {
+ 		"default_icon": "bitcoin.png",
+ 		"default_popup": "bitcoin.html"
+ 	}
+}
```

Comment:
Try adding this:

```json
"chrome_url_overrides": {
    "newtab": "bitcoin.html"
},
```

It might fix the issues with stuff not appearing in a new tab

Reply:
As long as your html file has the links to your styles and scripts, you can use the suggested method by @bryce-mcmath  to get the extension working correctly.